marko.json

```diff
@@ -1,4 +1,14 @@
 {
+    "<ebay-badge>": {
+        "renderer": "./src/components/ebay-badge/index.js",
+        "@*": "expression",
+        "@html-attributes": "expression",
+        "@class": "string",
+        "@style": "string",
+        "@type": "string",
+        "@number": "expression",
+        "@a11y-text": "string"
```

Comment:
The property is listed here as `a11y-text`, but the readme is asking for `aria-label`.

Reply:
Oh, I think I forgot to remove that. Let me get it fixed.